Sources/ViewInspector/Inspector.swift

```diff
@@ -98,8 +94,19 @@ public struct Inspector {
         return typeName
     }
 
-    private static var replacedGenericParametersCache: [String: [String: String]] = [:]
-    private static var sanitizedNamespacesCache: [String: String] = [:]
+    private final class Cache: @unchecked Sendable {
+        private let lock = NSLock()
+        var replacedGenericParameters: [String: [String: String]] = [:]
```

Comment:
IMO use private variables and make the internal accesses to this forcibly protected by the lock. 

Future maintainers may not realize that reading/writing to these cache values isn't thread safe. As written this type isn't actually Sendable, so it could cause issues.

Example of what I mean:
```swift
private var _replacedGenericParameters: [String: [String: String]] = [:]
var replacedGenericParameters: [String: [String: String]] {
    get { lock.protect { _replacedGenericParameters } }
    set { lock.protect { _ replacedGenericParameters = newValue } }
}
```

Reply:
Yeah, this is a valid concern. On the other hand, locking just inside getter and setter does not eliminate the issue completely, as the code that reads, modifies, and then writes back the value has to run under a single critical section (lock). If not, a concurrent flow can modify the value right after we read, so a subsequent write would not set the correct value already.